internal/pkg/debug/render/dot.go

```diff
@@ -0,0 +1,112 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package render
```

Comment:
Throughout:

- There are a lot of unhandled errors.  If you want to explicitly drop them, do `_ = r.WriteString(...)`
- Nit: The produced `.dot` should be at least partially human-readable.  Add indentation.
- There are three lingering `// TODO: document this somewhere?`

Reply:
Concerning unhandled errors: I've changed the code to swallow them explicitly. None of those calls can actually fail (`strings.Builder` always returns `len(s)` for the number of bytes and `nil` for the error).

I've added indentation to the `dot` rendering.

I've converted the TODO's to some documentation in the README.